### ISSUE TITLE
Which model should we choose for the next step of training?

### ISSUE THREAD

**Issue by longmalongma** [hkchengrex/Mask-Propagation#15]
Seven models were saved after pre-training with static images. Which model should we choose for the next step of training?

**Comment by hkchengrex** [comment]
Use the latest one without "checkpoint" in its name.

**Comment by longmalongma** [comment]
> Use the latest one without "checkpoint" in its name.

@hkchengrex 
![image](https://user-images.githubusercontent.com/40291722/115681184-df2ac780-a386-11eb-840b-50d34956a81b.png)

**Comment by longmalongma** [comment]
Apr21_17.49.37_retrain_s0_300000.pth?
Is that what you mean?

**Comment by hkchengrex** [comment]
Yes.

**Comment by longmalongma** [comment]
> Yes.

Ok, thanks.

**Comment by longmalongma** [comment]
> > Yes.
> 
> Ok, thanks.

@hkchengrex What is apr21_17.49.37_retrain_s0_checkpoint. PTH used for?

**Comment by hkchengrex** [comment]
When training is somehow interrupted and you want to resume (in the same stage).

**Comment by longmalongma** [comment]
> When training is somehow interrupted and you want to resume (in the same stage).

I sent you an email to discuss some details about saving the model with a training script.
